### DISCUSSION TITLE
Question 94: Can we add “Census-designated place” to rank for Political Details?  [RDLP-IZ]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#692] [Q&A] [answered]
Description: Per the US Census Bureau, CDPs are statistical equivalents of incorporated places and represent unincorporated communities that do not have a legally defined boundary or an active, functioning governmental structure.  Starting with the 2022 Boundary and Annexation Survey (BAS), tribal, state, and local governments can submit updates to Census Designated Places (CDPs). CDPs are a statistical geography representing closely settled, unincorporated communities that are locally recognized and identified by name. The purpose of CDPs is to provide meaningful statistics for well-known, unincorporated communities.

**Comment by rondlg** [comment, marked as answer]
Response:
	- SG to add to EMu when it’s back
	- Use abbreviation “CDP”

**Comment by rondlg** [comment]
Census-designated Place (CDP) - has been added to the "Political Classification Rank" lookup list.

- Summary string updates may be need so this issue added to structural changes list for discussion

**Comment by fmrdelapena** [comment]
I think we need to have a discussion with the larger group. i.e. the Sites working group.  There are issues to consider with the administrative hierarchy of City/Town, Village, CDP.  City/Town has its own table with ISO affiliation; Village/CDP does not. There can be villages nested within towns.  CDPs are not recognized as an "administrative" entity or division, but rather a populated area drawn out by the U.S. Census Bureau that are generally unincorporated and are not governed by a local municipality.  CDPs also don't have permanent boundaries like a city or town.  We need to better understand how structural changes will play into all of this, particularly how it will affect the summary string.
